packages/spectacle/package.json

```diff
@@ -84,36 +76,27 @@
   },
   "wireit": {
     "build": {
-      "dependencies": [
-        "build:lib",
-        "build:dist",
-        "types:create"
-      ]
+      "dependencies": [ "build:lib", "build:dist" ]
     },
     "build:lib": {
-      "dependencies": [
-        "build:lib:esm",
-        "build:lib:cjs"
-      ]
-    },
-    "build:lib:esm": {
-      "command": "nps babel:pkg:lib:esm",
+      "command": "pnpm tsup",
       "files": [
+        "./tsup.config.ts",
         "src/**",
         "!src/**/*.test.*",
         "../../package-scripts.js",
         "../../.babelrc.js",
         "../../.babelrc.build.js"
       ],
       "output": [
-        "es/**/*.js"
```

Comment:
Minor: You'll want to grep on `["/]es[/]` to catch a few more things still looking for `/es/`